src/main.rs

```diff
@@ -138,7 +149,11 @@ fn update_dependencies_impl(
 
     let new_content = cargo_toml.to_string();
     if new_content != cargo_toml_content {
-        Ok(Some(new_content))
+        if update {
+            return Ok(Some(new_content));
+        } else {
+            return Err("Dependencies are not up to date".into());
```

Comment:
```suggestion
            Err("Dependencies are not up to date".into())
```

Reply:
@0xsouravm probably some of these things get caught if you do `cargo clippy` and `cargo fmt`

I am guessing clippy would catch this kind of suggestion

Reply:
Yes

Reply:
Sorry about that! It's done now @shawntabrizi @Szegoo